experimental/Schemes/CoveredProjectiveSchemes.jl

```diff
@@ -281,7 +281,7 @@ function Base.show(io::IO, CPS::CoveredProjectiveScheme)
   n = length(projective_patches(CPS))
   K = base_ring(base_scheme(CPS))
   if get(io, :supercompact, false)
-    print(io, "Scheme")
+    print(io, "Covered projective scheme")
```

Comment:
In the code below this is referred to as either "Empty covered projective scheme" or as "Relative projective scheme" (the latter twice). What gives?

Reply:
It should be all "Relative projective scheme", thank you.